src/components/FormStepper.tsx

```diff
@@ -1,44 +1,48 @@
+import { useLocation } from 'react-router-dom';
 
 function FormStepper() {
-  
-    return (
+  const location = useLocation(); // Get the current location
 
-        <div>
-        <div className="steps-sidebar">
-          <div className="step active">
-            <div className="step-number">1</div>
-            <div className="step-label">
-              <span className="step-title">STEP 1</span>
-              <span className="step-name">YOUR INFO</span>
-            </div>
-          </div>
-  
-          <div className="step">
-            <div className="step-number">2</div>
-            <div className="step-label">
-              <span className="step-title">STEP 2</span>
-              <span className="step-name">SELECT PLAN</span>
-            </div>
+  // Determine the active step based on the current path
+  const activeStep = location.pathname; // You can customize this to fit your routes
+
+  return (
+    <div>
+      <div className="steps-sidebar">
+        <div className={`step ${activeStep === '/page1' ? 'active' : ''}`}>
+          <div className="step-number">1</div>
+          <div className="step-label">
+            <span className="step-title">STEP 1</span>
+            <span className="step-name">YOUR INFO</span>
           </div>
-  
```

Comment:
Extract the Step component with props of 
- is active
- title
- sub text
that way we will not be repeating ourselves so many times